feat/src/main/java/com/example/feat/home/components/ItemCourse.kt

```diff
@@ -93,7 +93,7 @@ fun ItemCourse(
                 .padding(5.dp)
         ){
             Text(
-                text = course.level?.lowercase(Locale.ROOT) ?: "Unknown",
+                text = course.level?.toExperienceText() ?: "Unknown",
```

Comment:
Magic text

Reply:
I have created this method to convert level to experience text .
and "unknown" is for avoid compile-error